cmd/storj-sim/prefix.go

```diff
@@ -54,7 +56,13 @@ func (writer *PrefixWriter) Prefixed(prefix string) io.Writer {
 	writer.prefixlen = max(writer.prefixlen, len(prefix))
 	writer.mu.Unlock()
 
-	return &prefixWriter{writer, prefix, "", make([]byte, 0, writer.maxline)}
+	return &prefixWriter{
+		PrefixWriter: writer,
+		prefix:       prefix,
+		local:        sync.Mutex{},
```

Comment:
just curious, do we need explicitly create new `sync.Mutex`?

Reply:
ah, not anymore, initially it wasn't using named fields, so yes... we can do without it